### PR TITLE
docs: update command line help in readme

### PR DESCRIPTION
**Pull request by andyredhead**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -83,6 +86,9 @@ OPTIONS:
             Sets write batch size
 ```
 
+The --schema-file option uses the same file format as --dry and --print-schema. Additional information about the format 
+may be gleaned by inspecting "fn from(json: &Value)" and "fn to_json(&self)" in the [Arrow DataType source code](https://github.com/apache/arrow-rs/blob/master/arrow/src/datatypes/datatype.rs).
```
Comment (domoritz) [inline Readme.md:90]:
```suggestion
```

File: Readme.md [hunk 2]
```diff
@@ -83,6 +86,9 @@ OPTIONS:
             Sets write batch size
 ```
 
+The --schema-file option uses the same file format as --dry and --print-schema. Additional information about the format
```
Comment (domoritz) [inline Readme.md:89]:
```suggestion
The --schema-file option uses the same file format as --dry and --print-schema.
```

**Review by domoritz — APPROVED** [review]
Please use semantic commit format for the pull request title. Something like docs: update command line help in readme

File: Readme.md (hunk 1 above)
Comment (andyredhead) [reply Readme.md:90]: If you think this is sufficient for end users, its fine by me. I fear some users may be left baffled. Lets see how it goes.

**Comment by domoritz** [comment]
Did you see my comment about the title?

**Comment by andyredhead** [comment]
No, unfortunately I hadn't spotted your instruction re the pull request title.

All 4 pull requests updated accordingly.

To aid future contributors I strongly suggest you document this requirement in the "for developers" section of the readme.

Looking back over the history of pull requests I can see that:
  - you modified the title of my previous pull request (fair enough, no issue with that)
  - there are lots of dependabot PRs in the format you requested
  - one or two other PRs from contributors where the title does not match the format you've specified

Perhaps this format is common across GitHub based open source projects, it is not common across the projects I've been involved with to date (there are lots of other weird & wonderful naming schemes, many of which reference work ticketing system ids etc.).

I don't have an issue with using a consistent format for things like pull request titles (or commit messages etc.). I do have a problem with the goalposts being moved while the game is in progress, especially when it seems I've been singled out for "special treatment".

**Comment by domoritz** [comment]
Thank you and thank you for raising the issue with contributing guidelines. I haven't really planned for contributions and appreciate you calling out what I need to do to make the repo welcoming for contributions.

**Comment by domoritz** [comment]
I added a contributing guide with some helpful pointers to make the expectations clearer: https://github.com/domoritz/csv2parquet/blob/main/CONTRIBUTING.md. I'd love to hear what you think. Again, thanks for calling this out as a need.

**Comment by andyredhead** [comment]
No worries, all good.

Had a read through the new contributions file, all makes perfect sense (and aligns with your previous requests :) ).

I've never used the word "reproductible", I'd use "reproducible".

It may be worth putting a link from the "For Developers" section on the readme over to the contributors page and possibly move the cargo clippy and fmt instruction into the contributors page (not a strong opinion, just putting all the "how things should be done" into one place).

Thank you for your time and the tools.

**Comment by domoritz** [comment]
Ups, typo. It should be reproducible. I think it's a great ideas to move the clippy instructions over. Do you want to take a stab at it?

**Comment by andyredhead** [comment]
Theoretically yes. Unfortunately a combination of a busy week at work and my sprog #2 just testing positive for covid (so is off school) means I'm unlikely to find time this week, sorry :(

If it still needs done next week I'll have a go.

**Comment by domoritz** [comment]
Oh no, I'm sorry to hear that. May they be well soon again. I pushed the changes just now so you can focus on your family.

**Pull request merged**